roxie/topo/toposerver.cpp

```diff
@@ -315,9 +315,9 @@ int main(int argc, const char *argv[])
         }
 
         // locate settings xml file in runtime dir
-        Owned<IPropertyTree> topology = loadConfiguration(defaultYaml, argv, "TopoServer", "TOPOSERVER", "toposerver.xml", nullptr);
+        Owned<IPropertyTree> topology = loadConfiguration(defaultYaml, argv, "roxie", "ROXIE", nullptr, nullptr);
```

Comment:
I'm not so sure about this change.  It may make sense if toposerver is only ever generated as a side-effect of deploying a roxie cluster.

Reply:
E.g. does this mean that roxie and toposerver they will think they have the same name?

Reply:
toposerver IS only deployed as a side effect of a roxie...